app/lib/mongo_exporter.rb

```diff
@@ -9,8 +9,8 @@ def self.export(collection:, path:)
       "mongoexport",
       "--uri=#{ENV['MONGODB_URI']}",
       "--collection=#{collection}",
-      "--out=#{File.join(path, [collection, 'json'].join('.'))}",
       "--type=json",
+      " | gzip > #{File.join(path, [collection, 'json', 'gz'].join('.'))}",
```

Comment:
optional: consider assigning the filename to a variable, to take the filename-constructing logic out of the string and improve readability.

Reply:
done